ZenPacks/zenoss/ZenPackLib/lib/helpers/utils.py

```diff
@@ -0,0 +1,256 @@
+import os
+import logging
+from collections import OrderedDict
+import yaml
+import time
+from .ZenPackLibLog import LOG
+from .Dumper import Dumper
+from .Loader import Loader
+
+
+# list of yaml sections with DEFAULTS capability
+YAML_HAS_DEFAULTS = ['classes', 'properties', 'thresholds', 'datasources',
+                     'datapoints', 'graphs', 'relationships','graphpoints']
+
+# preferred section ordering for YAML sections
+YAML_PREFERRED_ORDER = ['zProperties', 'class_relationships', 'classes',
+                        'device_classes', 'DEFAULTS', 'base',
+                        'label', 'order', 'properties',
+                        'monitoring_templates', 'dynamicview_views',
+                        'dynamicview_relations', 'impacts', 'impacted_by',
+                        'relationships']
+
+
+def load_yaml(yaml_doc=None, verbose=False, level=0):
+    ''''''
+    Loader.QUIET= not verbose
+    Loader.LEVEL = level
+
+    if not yaml_doc:
```

Comment:
The old version would default to using zenpack.yaml.  shouldn't we keep this ability?